sample-apps/spark/Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY ./settings.gradle ./settings.gradle
 
 RUN gradle build
 RUN tar -xvf build/distributions/spark-sample-app-1.0.tar
-RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v0.12.1-aws.1/aws-opentelemetry-agent.jar
+RUN wget https://github.com/aws-observability/aws-otel-java-instrumentation/releases/download/v0.18.0-aws.1/aws-opentelemetry-agent.jar
```

Comment:
Maybe we should make it an `ARG` in Dockerfile and give it a default value. In that case we can bump the version in tf files.
Does all otel sdk (for different languages) follows same version number? Or each language has its own?

Reply:
This is Java Auto Instrument Agent for Tracing. It is not OTel SDK so the version # is not in v1.0.x yet.

This sample app image doesn't change frequently yet. I like your `ARG` idea. I'll save it for later. :)